__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+const request = require('supertest');
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+  test('respond with 200 and return all topics', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics.length).toBe(3);
+        body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test('respond with 400 if endpoint is known', () => {
+    return request(app)
+      .get('/banana')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
```

Comment:
Great test - but since this doesn't apply to GET /api/topics exactly, (it's universal) I'd move it into it's own describe block